src/test/java/core/basesyntax/FruitStorageTest.java

```diff
@@ -0,0 +1,171 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThrows;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.fao.FileReaderImpl;
+import core.basesyntax.fao.FileReaderMy;
+import core.basesyntax.fao.FileWriterImpl;
+import core.basesyntax.fao.FileWriterMy;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.ReportGenerator;
+import core.basesyntax.model.ReportGeneratorImpl;
+import core.basesyntax.model.convertor.DataConvertor;
+import core.basesyntax.model.convertor.DataConvertorImpl;
+import core.basesyntax.model.handler.BalanceOperation;
+import core.basesyntax.model.handler.OperationHandler;
+import core.basesyntax.model.handler.PurchaseOperation;
+import core.basesyntax.model.handler.ReturnOperation;
+import core.basesyntax.model.handler.SupplyOperation;
+import core.basesyntax.model.strategy.OperationStrategy;
+import core.basesyntax.model.strategy.OperationStrategyImpl;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.ShopServiceImpl;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+/**
+ * Feel free to remove this class and create your own.
+ */
+
+public class FruitStorageTest {
```

Comment:
you should create separate test classes with the same structure as in the main directory. Also, check test coverage.

Reply:
not fixed